### ISSUE TITLE
How to change View center without triggering layer requests

### ISSUE THREAD

**Issue by nboisteault** [openlayers/openlayers#13574]
**Is your feature request related to a problem? Please describe.**
Hi, I'd like to know if it is possible to change a `View` center without automatically triggering layer requests (WMS, WMTS...).
I have tried `map.getView().set('center', [ lon, lat], true);` using the `opt_silent` parameter but the view is not moving at all.

**Describe the solution you'd like**
In OL2 you could use `setCenter` on a `Map` object with the `dragging` parameter set to `false`. Is there any equivalent in OL6? Thanks.

```js
setCenter: function(	lonlat,
zoom,
dragging, // {Boolean} Specifies whether or not to trigger movestart/end events
forceZoomChange	)
```

**Comment by ahocevar** [comment]
That's not possible. What is your use case that makes such a behavior necessary?

**Comment by nboisteault** [comment]
The code base of [Lizmap](https://github.com/3liz/lizmap-web-client) is very linked to OL2. As it would need alot of time to migrate to OL6 in one big step, we're trying small steps by migrating features one by one.
I had success by having an OL6 map behind the OL2 with their view synchronized https://github.com/3liz/lizmap-web-client/blob/master/assets/src/modules/Map.js#L47
We have migrated Overview map and Scaleline thanks to this hack for instance.
Now we'd like to migrate base layers to OL6. If I use OL2 `moveend` event to synchronize OL6, there is a shift between the two views when dragging. If I use OL2 `move` event, OL6 map follow OL2 map but all layers are requested at every tiny dragging change so it is not viable of course.
Here is where I hit a wall :)
Thanks for any suggestion.

**Comment by mike-000** [comment]
What type of layer requests are causing the problem?  Tile layers only request if new tiles are needed.  Vector layers by default do not update while the view is animating.  `setCenter()` and `setZoom()` will stop any outstanding interactions at each move, but if followed by a long duration very small change `animate()` an animating state could be maintained until the `moveend`.  https://codesandbox.io/s/vector-wfs-forked-h3wgio?file=/main.js

**Comment by nboisteault** [comment]
I'm using `ImageWMS`. As you can see in this GIF there are two requests made when moving top then bottom. I'd prefer to only have one request made when releasing mouse button like the third request made by OL2.
I have found that setting `ratio: 2` avoid those extra requests. This solution is good enough in my case. Does your solution with `animate()` would work with WMS? I'm not sure I have understood how it works.

![center_ol6](https://user-images.githubusercontent.com/2145040/164199958-099c12c9-8abd-4f6f-91f7-629c9fec5ced.gif)
.

**Comment by mike-000** [comment]
No, it will not prevent tiles being loaded as that option was deprecated https://github.com/openlayers/openlayers/blob/main/changelog/upgrade-notes.md#always-load-tiles-while-animating-or-interacting so it is the same behaviour as you would see if you were panning the OL6 map yourself.

**Comment by ahocevar** [comment]
@nboisteault What @mike-000 said only applies to tile layers. In your case the layer is an image layer, which usually does not load new images during panning. You can see that in https://openlayers.org/en/latest/examples/wms-image.html. I'm surprised to see the requests in your example. Looks like you must have some custom loading behavior that triggers this.

**Comment by mike-000** [comment]
> @nboisteault What @mike-000 said only applies to tile layers. In your case the layer is an image layer, which usually does not load new images during panning. You can see that in https://openlayers.org/en/latest/examples/wms-image.html. I'm surprised to see the requests in your example. Looks like you might have some custom loading behavior that triggers this.

Yes. a standard ImageWMS source behaves in a similar way to the previous vector source with bbox strategy example when an animating state is maintained https://codesandbox.io/s/wms-image-forked-k7xcjz?file=/main.js
